libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -286,10 +286,9 @@ constructor(
         mainJobController.scope.launch {
             Log.d(TAG, "onDestroy")
             MapboxNavigationTelemetry.unregisterListeners(this@MapboxNavigation)
-            ThreadController.cancelAllNonUICoroutines()
-            ThreadController.cancelAllUICoroutines()
```

Comment:
inside tripSession.stop already cancel all the coroutines jobs.

Reply:
The ones canceled in `MapboxTripSession#stop` are local to `MapboxTripSession` i.e. we should leave these as they're stopping the "root" ones. Also I think even if they were already canceled calling these wouldn't be harmful, good to be precautious 😅

Reply:
it's reverted.